src/test/java/uk/gov/companieshouse/certifiedcopies/orders/api/controller/CertifiedCopiedControllerIntegrationTest.java

```diff
@@ -0,0 +1,157 @@
+
+    @Autowired
+    private CertifiedCopyItemRepository repository;
+
+    @AfterEach
+    void tearDown() {
+        repository.deleteAll();
+    }
+
+    @Test
+    @DisplayName("Successfully creates certified copy item")
+    void createCertifiedCopyItemSuccessfullyCreatesCertifiedCopyItem() throws Exception {
+        final CertifiedCopyItemRequestDTO certifiedCopyItemDTORequest = new CertifiedCopyItemRequestDTO();
+        certifiedCopyItemDTORequest.setCompanyNumber(COMPANY_NUMBER);
+        certifiedCopyItemDTORequest.setCustomerReference(CUSTOMER_REFERENCE);
+        certifiedCopyItemDTORequest.setQuantity(QUANTITY);
+
+        final FilingHistoryDocumentRequestDTO filingHistoryDocumentRequestDTO = new FilingHistoryDocumentRequestDTO();
+        filingHistoryDocumentRequestDTO.setFilingHistoryId(FILING_HISTORY_ID);
+
+        final CertifiedCopyItemOptionsRequestDTO certifiedCopyItemOptionsDTORequest = new CertifiedCopyItemOptionsRequestDTO();
+        certifiedCopyItemOptionsDTORequest.setContactNumber(CONTACT_NUMBER);
+        certifiedCopyItemOptionsDTORequest.setDeliveryMethod(DeliveryMethod.POSTAL);
+        certifiedCopyItemOptionsDTORequest.setDeliveryTimescale(DeliveryTimescale.STANDARD);
+        certifiedCopyItemOptionsDTORequest.setForename(FORENAME);
+        certifiedCopyItemOptionsDTORequest.setSurname(SURNAME);
+
+        certifiedCopyItemOptionsDTORequest.setFilingHistoryDocuments(Arrays.asList(filingHistoryDocumentRequestDTO));
+        certifiedCopyItemDTORequest.setItemOptions(certifiedCopyItemOptionsDTORequest);
+
+        mockMvc.perform(post(CERTIFIED_COPIED_URL)
+                .header(REQUEST_ID_HEADER_NAME, REQUEST_ID_VALUE)
+                .header(ERIC_IDENTITY_TYPE, ERIC_IDENTITY_TYPE_OAUTH2_VALUE)
+                .header(ERIC_IDENTITY, ERIC_IDENTITY_VALUE)
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(objectMapper.writeValueAsString(certifiedCopyItemDTORequest)))
+                .andExpect(status().isCreated())
+                .andExpect(jsonPath("$.company_number", is(COMPANY_NUMBER)))
+                .andExpect(jsonPath("$.customer_reference", is(CUSTOMER_REFERENCE)))
+                .andExpect(jsonPath("$.item_options.contact_number", is(CONTACT_NUMBER)))
+                .andExpect(jsonPath("$.item_options.delivery_method", is(DeliveryMethod.POSTAL.getJsonName())))
+                .andExpect(jsonPath("$.item_options.delivery_timescale", is(DeliveryTimescale.STANDARD.getJsonName())))
+                .andExpect(jsonPath("$.item_options.filing_history_documents[0].filing_history_id", is(FILING_HISTORY_ID)))
+                .andExpect(jsonPath("$.item_options.forename", is(FORENAME)))
+                .andExpect(jsonPath("$.item_options.surname", is(SURNAME)))
+                .andExpect(jsonPath("$.kind", is(KIND)))
+                .andExpect(jsonPath("$.postal_delivery", is(true)))
+                .andExpect(jsonPath("$.quantity", is(QUANTITY)));
```

Comment:
Tests such as this could be made to check the item is successfully created in the DB too?

Reply:
Good point, I'll add one in

Reply:
Done